src/modules/about-us/OurMission/LeftSide.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+
+type LeftSideProps = {
+  title: string;
+  description: string;
+  imageSrc: string;
+};
+
+const LeftSide = ({ title, description, imageSrc }: LeftSideProps) => (
+  <div className="relative flex flex-col-reverse gap-2 lg:flex-row lg:gap-4">
+    <div className="relative h-32 overflow-hidden lg:w-3/5">
+      <div className="absolute -left-8 h-full w-full skew-x-[20deg] bg-primary-600"></div>
+      <div className="relative h-full px-20">
+        <div className="flex h-full flex-col justify-center gap-2">
+          <h4 className="non-italic text-white">{title}</h4>
+          <p className="text-stone-300">{description}</p>
+        </div>
+      </div>
+    </div>
+    <div className="flex justify-center">
+      <Image
+        src={imageSrc}
+        alt="community engagement"
+        className="h-36 content-center lg:h-48"
```

Comment:
@sonam-serchan Does this Image component work without `height` and `width` props? I think they are both required unless you have `fill` property

Reply:
Hey @titungdup, yeah it seems it works with tailwind class. It probably hasn't been covered in the docs.

The closest I have found is with `style` in the doc https://nextjs.org/docs/pages/api-reference/components/image#style

But here's the related info.
![Screenshot_20240627_080351_Chrome.jpg](https://github.com/xplorer-io/website/assets/23254380/7064a85f-70ea-4bb8-a6f7-f30eccf2e3c0)

Reply:
@sonam-serchan all got then 👍